docs/charts.md

```diff
@@ -204,23 +205,25 @@ team.
 
 ## Templates and Values
 
-By default, Helm Chart templates are written in the Go template language, with the
+By default, Helm Chart templates are written in the
```

Comment:
Wondering if it makes sense to drop the `By default` for now, since it implies there are other templating engines. Not a big deal, but could mislead.